uPortal-security/uPortal-security-mvc/src/main/java/org/apereo/portal/security/mvc/LogoutController.java

```diff
@@ -43,15 +40,19 @@
  */
 @Controller
 @RequestMapping("/Logout")
-public class LogoutController implements InitializingBean {
+public class LogoutController {
+
+    /* package-private */ static final String LOGOUT_REDIRECT_PREFIX = "logoutRedirect.";
+    /* package-private */ static final String LOGOUT_REDIRECT_ROOT = "root";
 
-    private static final Log log = LogFactory.getLog(LogoutController.class);
+    private final Logger logger = LoggerFactory.getLogger(getClass());
```

Comment:
:art: It would be nice to use [`@Slf4j`](https://projectlombok.org/features/log)